include/Metabox/Posts/Tag.php

```diff
@@ -68,23 +68,24 @@ public function move( $options ) {
 			'tag__in'   => $options['old_tag'],
 			'post_type' => 'post',
 			'nopaging'  => 'true',
-		));
+		) );
 
 		foreach ( $posts as $post ) {
-            $current_tags = wp_get_post_tags( $post->ID, array( 'fields' => 'ids' ) );
-            $current_tags = array_diff( $current_tags, array( $options['old_tag'] ) );
+			$current_tags = wp_get_post_tags( $post->ID, array( 'fields' => 'ids' ) );
```

Comment:
_Comment from Scrutinizer. Looks like an existing issue though_

It seems like $current_tags can also be of type WP_Error; however, parameter $array1 of array_diff() does only seem to accept array, maybe add an additional type check?